### ISSUE TITLE
demo program

### ISSUE THREAD

**Issue by fvega99** [ROBOTIS-OP/robotis_op_framework#4]
How can I run the demo program through ros ?

**Comment by kdaun** [comment]
What kind of demo program do you mean? The basic OP2 demo program that is shipped with the robot?

**Comment by fvega99** [comment]
Yes that demo program .i need to add a subscriber in order to control it with a joystick .

**Comment by kdaun** [comment]
For controlling the OP2 with a joystick you can do it similar to the XBOX 360 controller with a different key mapping, see [robotis_op_teleop_xbox360_controller](https://github.com/ROBOTIS-OP/robotis_op_teleop/blob/master/src/robotis_op_teleop_xbox360_controller.cpp).
For performing the demo motions, you can use the [action interface](https://github.com/ROBOTIS-OP/robotis_op_ros_control/blob/master/include/robotis_op_ros_control/robotis_op_ros_controller.h#L42), an example is the kick motion in the [keyboard teleop](https://github.com/ROBOTIS-OP/robotis_op_teleop/blob/master/src/robotis_op_teleop_keyboard.cpp#L316).

**Comment by fvega99** [comment]
thanks for the help kevin. Is there a way to go directly to a specific demo
program. like when the demo program is run, it directly goes to soccer mode
and starts playing it ?
